src/extensions/cellpack/state.ts

```diff
@@ -94,4 +96,113 @@ const StructureFromCellpack = PluginStateTransform.BuiltIn({
             }
         }
     }
-});
\ No newline at end of file
+});
+
+export { GetAllAssamblyinOne };
```

Comment:
this one is not used, right?

Reply:
Its not, I wasnt sure if it was best to do it from the Model or the Structure. You can also change the name if you want.